composer.json

```diff
@@ -31,12 +31,11 @@
             "email": "guilhermeblanco@hotmail.com"
         }
     ],
-    "minimum-stability": "alpha",
     "require":           {
         "php":                               ">=5.4",
-        "doctrine/doctrine-module":          "0.9.*",
-        "doctrine/orm":                      ">=2.4,<2.7-dev",
-        "doctrine/dbal":                     ">=2.4,<2.7-dev",
+        "doctrine/doctrine-module":          ">=0.8",
```

Comment:
Ok the correct flow is 
* Before this release, release of DoctrineModule 1.0
* This module dep from ~1.0